### DISCUSSION TITLE
Federation example that use subgraphs from Apollo Federation documentation

### DISCUSSION THREAD

**Discussion by vinh0604** [Netflix/dgs-framework#554] [Show and tell]
Hi DGS community,
I was looking for some examples that demonstrate the reference resolver that is mentioned here https://www.apollographql.com/docs/federation/entities/#resolving, but I couldn't find any.
So I tinkered a bit to create a Federation example that uses the same subgraphs mentioned in the Apollo doc, which I hope it'll useful for others as it shows how reference resolver is implemented with DGS and also includes examples for `@provides` `@requires` directives.

https://github.com/vinh0604/netflix-dgs-federation-examples

I want to know if this example project can be included in the community example section of the docs as well.

Cheers.

**Comment by paulbakker** [comment]
We have an example with Apollo federation here: https://github.com/Netflix/dgs-federation-example. 

I like the idea of a community section though (with your example), I'll add that!
